ucm2/Qualcomm/sdm845/HiFi-MM1.conf

```diff
@@ -0,0 +1,78 @@
+# Use case configuration for LenovoYOGAC6301.
+# Author: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
+
+SectionVerb {
+
+	EnableSequence [
+		cset "name='SLIMBUS_0_RX Audio Mixer MultiMedia1' 1"
+		cset "name='MultiMedia2 Mixer SLIMBUS_0_TX' 1"
+	]
+
+	Include.wcde.File "/codecs/wcd934x/DefaultEnableSeq.conf"
+	Include.wsae.File "/codecs/wsa881x/DefaultEnableSeq.conf"
+
+	Include.wcdd {
+		File "/codecs/wcd934x/DefaultDisableSeq.conf"
+		Before.DisableSequence "0"
+	}
+
+	DisableSequence [
+		cset "name='SLIMBUS_0_RX Audio Mixer MultiMedia1' 0"
+		cset "name='MultiMedia2 Mixer SLIMBUS_0_TX' 0"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker playback"
+
+	Include.wcde.File "/codecs/wcd934x/SpeakerEnableSeq.conf"
+	Include.wsae.File "/codecs/wsa881x/SpeakerEnableSeq.conf"
+	Include.wsad.File "/codecs/wsa881x/SpeakerDisableSeq.conf"
+	Include.wcdd.File "/codecs/wcd934x/SpeakerDisableSeq.conf"
+
+	Value {
+		PlaybackPriority 100
+		PlaybackPCM "hw:${CardId}"
+		PlaybackVolume "RX2 Digital Volume"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Headphones playback"
+
+	ConflictingDevice [
+		"Speaker"
+	]
+
+	Include.wcdhpe.File "/codecs/wcd934x/HeadphoneEnableSeq.conf"
+	Include.wcdhpd.File "/codecs/wcd934x/HeadphoneDisableSeq.conf"
+
+	Value {
+		PlaybackPriority 200
+		PlaybackPCM "hw:${CardId}"
+		PlaybackVolume "RX1 Digital Volume"
+		PlaybackVolume "RX2 Digital Volume"
+	}
+}
+
+SectionDevice."Mic" {
+	Comment "Mic"
+
+	Include.wcdmice.File "/codecs/wcd934x/HeadphoneMicEnableSeq.conf"
+	Include.wcdmicd.File "/codecs/wcd934x/HeadphoneMicDisableSeq.conf"
+
+	DisableSequence [
+		cset "name='AMIC MUX0' ZERO"
+		cset "name='ADC2 Volume' 0"
```

Comment:
HW volume should not be set in enable/disable sequences. The application is expected to initialize this volume for the normal operation. Move this to the boot sequence (look for `init.conf` files in codecs/).

Reply:
will do.